cg/cli/deliver/base.py

```diff
@@ -183,5 +187,39 @@ def deliver_old_analysis(context, case_id: str, ticket_id: int, delivery_type: s
         deliver_api.deliver_files(case_obj=case_obj)
 
 
+@click.command(name="rsync")
+@click.option(
+    "-t", "--ticket-id", type=int, help="Rsync the files for a specific ticket", required=True
+)
+@click.option("--dry-run", is_flag=True)
+@click.option("--covid", is_flag=True)
+@click.pass_context
+def rsync(context, ticket_id: int, dry_run: bool, covid: bool):
+    """The folder generated using the "cg deliver analysis" command will be
+    rsynced with this function to the customers inbox on caesar.
+    """
+    inbox = context.obj.get("delivery_path")
+    status_db = context.obj["status_db"]
+    rsync_api = RsyncAPI(store=status_db)
+    cmd = rsync_api.generate_rsync_command(ticket_id=ticket_id, base_path=str(inbox))
+    if dry_run:
+        LOG.info("Dry-run activated, skipping running the command: %s", cmd)
+    else:
+        LOG.info("Command ran: {}".format(cmd))
+        proc = subprocess.Popen(cmd.split())
```

Comment:
I suggest to use Process from cg.utils for this instead!

Reply:
Yes please use that